index.html

```diff
@@ -2506,7 +2517,7 @@ <h3>Data Schemas</h3>
 In the example above, the <a>issuer</a> is specifying a
 <code>credentialSchema</code> pointing to a means of transforming the input
 data into a format which can then be used by a <a>verifier</a> to determine if
-the proof provided with the <a>verifiable credential</a> is valid.
+the proof provided with the <a>verifiable credential</a> is well-formed.
```

Comment:
what is the difference between a "well-formed" and a "conforming document".

Reply:
excellent, I defined the term here: https://github.com/w3c/vc-data-model/pull/1320

We use well formed in a few other places, so it will be helpful to define the term once.

Reply:
But that's not the definition that applies to this use.

Well-formed does not mean conformant to the VCDM. Well-formed in this usage means the shape of the data matches that specified by the value of the credentialSchema property.